packages/Config/RectorConfig.php

```diff
@@ -169,6 +169,7 @@ public function rule(string $rectorClass): void
     public function rules(array $rectorClasses): void
     {
         Assert::allString($rectorClasses);
+        Assert::uniqueValues($rectorClasses);
```

Comment:
Ultimate solution to duplicated in-set rules :)

Ref https://github.com/rectorphp/rector-src/pull/4381